worlds/ufo50/options.py

```diff
@@ -0,0 +1,52 @@
+from dataclasses import dataclass
+from Options import (DefaultOnToggle, Toggle, StartInventoryPool, Choice, Range, OptionSet, PerGameCommonOptions,
+                     OptionGroup)
+from .constants import game_ids
+
+
+class AlwaysOnGames(OptionSet):
+    """
+    Choose which games you would like to enable.
+    """
+    internal_name = "always_on_games"
+    display_name = "Always On Games"
+    valid_keys = {game_name for game_name in game_ids.keys()}
+
+
+class RandomChoiceGames(OptionSet):
+    """
+    Choose which games have a chance of being enabled.
+    The number of games that will be enabled is based on the Random Choice Game Count option.
+    """
+    internal_name = "random_choice_games"
+    display_name = "Random Choice Games"
+    valid_keys = {game_name for game_name in game_ids.keys()}
+
+
+class RandomChoiceGameCount(Range):
+    """
+    Choose how many Random Choice Games will be included alongside your Always On Games.
+    If you do not have enough Random Choice Games selected, it will enable all of them.
```

Comment:
```suggestion
    If you do not have enough Random Choice Games selected, it will enable all your selected games.
```
I think this is otherwise just about ambiguous enough that somebody would be confused over it at some point.